common/app/com/commercetools/sunrise/common/utils/ErrorFormatter.java

```diff
@@ -18,16 +21,22 @@
      * @param message error message
      * @return the error message localized and formatted
      */
-    String format(final List<Locale> locales, final String message);
+    String format(final List<Locale> locales, final String message, final Map<String, Object> hashArgs);
```

Comment:
I would prefer the parameter name `args` here.

Reply:
I named them `hashArgs` here and in `I18nResolver` because they are hash arguments. What don't you like in particular from the name?

Reply:
Done!